frontend/package.json

```diff
@@ -150,7 +150,7 @@
     "null-loader": "^3.0.0",
     "openshift-logos-icon": "1.7.1",
     "patternfly": "^3.59.5",
-    "patternfly-react": "2.39.16",
+    "patternfly-react": "2.39.17",
```

Comment:
I think this is the older PF 3 react package. The new one exists in `@patternfly/react-core`.

Reply:
Seems like we are using the latest available version then.

Reply:
Updated `@patternfly/react-core` to 4.97.2, which includes the fix: https://www.patternfly.org/v4/developer-resources/release-notes/html